app/services/force/listing_service.rb

```diff
@@ -180,11 +184,8 @@ def self.array_sort!(listing)
       # Add stubbed listing fields
       # rubocop:disable LineLength
       stubbed_listing_data = {
-        'Allows_Realtor_Commission' => true,
-        'Realtor_Commission_Percentage' => 15,
-        'Realtor_Commission_Info' => 'TBD but this will probably be a shortish string',
         'CC_and_R_URL' => 'http://www.google.com',
-        'Repricing_Mechanism' => 'TODO: Replace this with a real example of a repricing mechanism. Here\'s some sample text with links <a href=\"http://sf-moh.org/index.aspx?page=295\" target=\"_blank\">Inclusionary Affordable Housing Program Monitoring and Procedures Manual 2013</a>',
+        'Repricing_Mechanism' => 'TODO: Replace this with a real example of a repricing mechanism. Here\'s some sample text with links <a href="http://sf-moh.org/index.aspx?page=295" target="_blank">Inclusionary Affordable Housing Program Monitoring and Procedures Manual 2013</a>',
```

Comment:
uh oh, does this mean that this field is missing?

Reply:
nevermind, I see that you were just changing the formatting. I went ahead and removed the fields from this PR that we're getting from salesforce now.